### PR TITLE
2.4 Support section updates

### PR DESCRIPTION
**Pull request by zkayyali812**

### CODE REVIEW

File: stories/assembly-aws-support.adoc [hunk 1]
```diff
@@ -5,41 +5,14 @@ ifdef::context[:parent-context: {context}]
...
 
-{AAPonAWS} is a _self-managed_ deployment. 
+{AAPonAWS} is a self-managed deployment. 
 When the application is deployed into your AWS infrastructure, customers are responsible for the maintenance of the AWS infrastructure, OS patching, and {PlatformNameShort} patching.  
 
 Red Hat does not support changes to the infrastructure resources deployed as part of the solution unless there is documentation by Red Hat describing the process, such as in route table configuration for networking or documented upgrade processes. 
 Adding additional compute resources outside of extension nodes voids Red Hat support for {AAPonAWS} deployment.
 
-Upgrades to {PlatformNameShort} from {Market} will be performed differently from self-installed {PlatformNameShort}. 
+Upgrades to {AAPonAWS} will be performed differently from self-installed {PlatformNameShort}.
```
Comment (ianf77) [inline stories/assembly-aws-support.adoc:14]: Upgrades to {AAPonAWS} are performed differently...

File: stories/assembly-gcp-support.adoc [hunk 2]
```diff
@@ -6,9 +6,14 @@ ifdef::context[:parent-context: {context}]
...
-When the application is deployed into your GCP infrastructure, customers are responsible for the maintenance of the GCP infrastructure and {PlatformNameShort} patching. 
-However, Red Hat does not support any changes to the infrastructure resources deployed as part of the solution unless there is documentation by Red Hat to do so, such as in route table configuration for networking or documented upgrade processes. 
-Adding additional compute resources outside of extension nodes voids Red Hat support for the {AAPonGCP} deployment.
+When the application is deployed into your GCP infrastructure, customers are responsible for the maintenance of the GCP infrastructure, OS patching, and {PlatformNameShort} patching. 
+
+Red Hat does not support changes to the infrastructure resources deployed as part of the solution unless there is documentation by Red Hat describing the process, such as in route table configuration for networking or documented upgrade processes. 
+Adding additional compute resources outside of extension nodes voids Red Hat support for {AAPonGCP} deployment.
+
+Upgrades to {AAPonGCP} will be performed differently from self-installed {PlatformNameShort}.
```
Comment (ianf77) [inline stories/assembly-gcp-support.adoc:14]: Upgrades to {AAPonGCP} are performed...

File: stories/topics/ref-gcp-supported-changes.adoc [hunk 3]
```diff
@@ -11,18 +11,20 @@ Red Hat supports changes to the following options:
...
 .Red Hat Responsibilities
-Red Hat has the following responsibilities.
 
-* Premium support for {PlatformNameShort}.
+Red Hat has the following responsibilities:
+
+* Premium support for {PlatformNameShort}
 * Directions and how-to processes for upgrading {AAPonGCP}.
-** Provided through knowledge base articles when upgrades are published.
+* Red Hat will support the current version of {PlatformNameShort}. Bug fixes and CVE patches will require an upgrade to the latest version
```
Comment (ianf77) [inline stories/topics/ref-gcp-supported-changes.adoc:19]: Red Hat supports the current version of {PlatformNameShort}   

(does this mean previous versions are not supported?)

Bug fixes and CVE patches require an upgrade to the latest version.

**Review by ianf77 — COMMENTED** [review]
A few changes.

File: stories/topics/ref-gcp-supported-changes.adoc (hunk 3 above)
Comment (zkayyali812) [reply stories/topics/ref-gcp-supported-changes.adoc:19]: This is meaning that bug fixes and CVE patches are fixed in the next released version of Self managed AoC

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**